src/components/__tests__/TestUtils.ts

```diff
@@ -134,4 +144,16 @@ describe("TestUtils", () => {
     let updatedLibrary = modifyLibrary(baseLibrary, {uuid: "UUID2"});
     expect(updatedLibrary.uuid).to.equal("UUID2");
   });
+
+  it("validates the contact email", () => {
+    let baseLibrary = testLibrary1;
+    expect(baseLibrary.urls_and_contact.contact_validated).to.be.undefined;
+    baseLibrary = validate(baseLibrary);
+    expect(baseLibrary.urls_and_contact.contact_validated).to.equal(baseLibrary.basic_info.timestamp);
+    baseLibrary = validate(baseLibrary, "Fri, 12 May 2018 15:05:34 GMT");
+    expect(baseLibrary.urls_and_contact.contact_validated).to.equal("Fri, 12 May 2018 15:05:34 GMT");
+    let noTimestamp = {...baseLibrary.basic_info, ...{timestamp: undefined}};
```

Comment:
You should be able to do `let noTimestamp = {...baseLibrary.basic_info, timestamp: undefined };` instead of having the second argument in a destructured object. Same for line 94 above.